cw_core/lib/address_info.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:hive/hive.dart';
+
+part 'address_info.g.dart';
+
+@HiveType(typeId: AddressInfo.typeId)
+class AddressInfo extends HiveObject {
+  AddressInfo({required this.address, this.accountIndex, required this.label});
+
+  static const typeId = 11;
```

Comment:
@konstantinullrich when this PR is merged, please also add this `typeId` to the new approach of saving type ids

Reply:
Please merge #1011 that introduces the new `typeId` Approach